src/components/VSelectField/VSelectField.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div
+    class="relative m-0.5px box-content block w-fit overflow-hidden rounded-sm border text-sm focus-within:m-0 focus-within:border-1.5 focus-within:border-pink hover:border-dark-charcoal focus-within:hover:border-pink"
+    :class="variant === 'borderless' ? 'border-tx' : 'border-dark-charcoal-20'"
+  >
+    <div class="pointer-events-none absolute inset-y-0 my-auto h-fit start-2">
+      <slot name="start" />
+    </div>
+    <div class="pointer-events-none absolute inset-y-0 my-auto h-fit end-2">
+      <VIcon :icon-path="caretDownIcon" />
+    </div>
+    <select
+      :id="fieldId"
+      v-model="modelMedium"
+      class="flex h-[calc(theme(spacing.10)_-_2_*_theme(borderWidth.DEFAULT))] w-full appearance-none bg-white pe-10"
```

Comment:
```suggestion
      class="flex h-[calc(theme(spacing.10)_-_2_*_theme(borderWidth.DEFAULT))] w-full appearance-none bg-white pe-10 truncate"
```

I would recommend we truncate for improved styling for long languages:

|before|after|
|-|-|
|<img width="241" alt="CleanShot 2022-09-21 at 20 37 21@2x" src="https://user-images.githubusercontent.com/6351754/191633678-a895cb77-d03c-41f5-a477-ccf40f4c7a59.png">|<img width="249" alt="CleanShot 2022-09-21 at 20 37 09@2x" src="https://user-images.githubusercontent.com/6351754/191633707-a96e8504-c921-492f-92a3-28779a7a835d.png">|